homeassistant/components/iqvia/sensor.py

```diff
@@ -104,7 +104,11 @@ class ForecastSensor(IQVIAEntity):
     @callback
     def update_from_latest_data(self):
         """Update the sensor."""
+        if not self.coordinator.last_update_success:
```

Comment:
Instead of this, you should update `IQVIAEntity._handle_coordinator_update` to not call `update_from_latest_data`